stac_ipyleaflet/stac_discovery/stac_widget.py

```diff
@@ -1,77 +1,133 @@
-            # {"name": "MAAP STAC", "url": "https://wssn144yw1.execute-api.us-west-2.amazonaws.com/"},            
-            {"name": "Element84 Earth Search", "url": "https://earth-search.aws.element84.com/v1"},
+            # {"name": "MAAP STAC", "url": "https://wssn144yw1.execute-api.us-west-2.amazonaws.com/"},
+            {
+                "name": "Element84 Earth Search",
+                "url": "https://earth-search.aws.element84.com/v1",
+            },
             # {"name": "Microsoft Planetary Computer", "url": "https://planetarycomputer.microsoft.com/api/stac/v1"},
         ]
         # make list of name values from stac_catalogs
-        catalog_options = sorted([c['name'] for c in stac_catalogs])
-        selected_catalog = stac_catalogs[0]
+        catalog_options = sorted([c["name"] for c in stac_catalogs])
         for cat in stac_catalogs:
-            # print(cat)
             stac_client = Client.open(cat["url"], headers=[])
             collections_object = stac_client.get_all_collections()
             collections = Stac.organize_collections(collections_object)
             cat["collections"] = collections
+        # set default catalog based on name
+        selected_catalog = [cat for cat in stac_catalogs if "MAAP" in cat["name"]][0]
+
         if "collections" not in selected_catalog:
-            print("COLLECTIONS NOT FOUND")
-            return None
-        # else:
-        selected_collection_options = sorted([c for c in selected_catalog["collections"]], key=lambda c: c["id"])
+            logging.warn("NO COLLECTIONS FOUND")
+
+        collections_filter_checkbox = Checkbox(
+            value=True, layout=layouts["checkbox"], indent=False
+        )
+
+        # available collections have been tagged as `has_cog: True` when reviewing item_assets
+        def get_available_collections(catalog):
+            if collections_filter_checkbox.value:
+                return sorted(
+                    [c for c in catalog["collections"] if c["has_cog"]],
+                    key=lambda c: c["id"],
+                )
+            else:
+                return sorted(
+                    [c for c in catalog["collections"]], key=lambda c: c["id"]
+                )
+
+        selected_collection_options = get_available_collections(
+            catalog=selected_catalog
+        )
         selected_collection = selected_collection_options[0]
-        self.stac_data = { 
+        self.stac_data = {
             "catalog": selected_catalog,
             "collection": selected_collection,
             "items": [],
-            "layer_added": False
+            "layer_added": False,
         }
 
         # STAC Widget Items
         catalogs_dropdown = Dropdown(
             options=catalog_options,
             value=self.stac_data["catalog"]["name"],
             style=styles["init"],
-            disabled=True,
+            # disabled=True,
```

Comment:
⛏️ : Is there a reason we would want to comment this out instead of removing? We should remove if it is not being used